torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -2198,7 +2564,7 @@ Error PyTorchModelLoader::loadAdaptiveAvgPool2d(
 
   size_t inputH = input.dims()[1];
   size_t inputW = input.dims()[2];
-
+  input = rescaleUIntToInt(input);
```

Comment:
thanks for adding this, can you put it right after the ASSIGN_VALUE_OR_RETURN_ERR so it doesn't get separate further, we had a bug earlier because of that